frappe/utils/data.py

```diff
@@ -1133,3 +1133,17 @@ def is_subset(list_a, list_b):
 
 def generate_hash(*args, **kwargs):
 	return frappe.generate_hash(*args, **kwargs)
+
+
```

Comment:
Our convention is to leave 1 line after a function... and 2 lines after a Class.

Reply:
1 blank line was causing deepsource analysis to fail hence I have added 2 lines.